tests/unit/stream_alert_cli/test_config.py

```diff
@@ -14,67 +14,73 @@
 limitations under the License.
 """
 # pylint: disable=protected-access
-from contextlib import nested
+# pylint: disable=no-self-use
```

Comment:
sorry one last nit - these can be on one line separated by commas :)

`pylint: disable=protected-access,no-self-use`

Reply:
Ah, yep, sure thing. didn't realise you can do this.

Reply:
Actually, since I'm now using self in the methods I can just remove it.